torch/csrc/jit/mobile/promoted_prim_ops.cpp

```diff
@@ -134,7 +135,7 @@ void toList(Stack& stack) {
   // Wrap out_ty in a ListType dim times.
   for (const auto i : c10::irange(dim_val)) {
     (void)i; // Suppress unused variable warning
-    out_ty = ListType::create(out_ty);
+    out_ty = at::ListType::create(out_ty);
```

Comment:
`DynamicTypeFactory::create<ListType>(out_ty);`
DynamicTypeFactory's header is ATen/core/type_factory.h